Perl/Perl.sublime-syntax

```diff
@@ -11,1092 +10,1046 @@ file_extensions:
   - PL
 first_line_match: ^#!.*\bperl\b
 scope: source.perl
+
+variables:
+  break: (?!\w| *::)
```

Comment:
presumably this space should be enclosed in a char class, else it could break as soon as this variable is used in expressions with extended mode enabled

Reply:
Which is actually the case in the `pragma` context without negative impact.

![screenshot](https://user-images.githubusercontent.com/16542113/41670139-bd19ebe4-74b3-11e8-9ebc-7125daf13091.png)

Reply:
... but there is another reason, why a change might be useful. The current solution doesn't respect tabs.